relay-server/src/services/processor.rs

```diff
@@ -282,7 +282,7 @@ pub enum ProcessingGroup {
 
 impl ProcessingGroup {
     /// Splits provided envelope into list of tuples of groups with associated envelopes.
-    fn split_envelope(
+    pub fn split_envelope(
```

Comment:
This shouldn't be necessary.

Reply:
We make use of it in `handle_process_envelope` in `proxy_processor.rs` hence I added this, but there might be a better way to go about this.

Reply:
Ok fair enough